src/main/java/seedu/taassist/logic/commands/EditCommand.java

```diff
@@ -103,10 +92,9 @@ private static Student createEditedStudent(Student studentToEdit, EditStudentDes
         Phone updatedPhone = editStudentDescriptor.getPhone().orElse(studentToEdit.getPhone());
         Email updatedEmail = editStudentDescriptor.getEmail().orElse(studentToEdit.getEmail());
         Address updatedAddress = editStudentDescriptor.getAddress().orElse(studentToEdit.getAddress());
-        Set<ModuleClass> updatedModuleClasses = editStudentDescriptor.getModuleClasses()
-                .orElse(studentToEdit.getModuleClasses());
+        List<StudentModuleData> updatedModuleData = studentToEdit.getModuleData(); // not editable
```

Comment:
```suggestion
        List<StudentModuleData> moduleData = studentToEdit.getModuleData(); // not editable
```
Perhaps do it as so to clarify that it isn't editable.